apps/frontpage/scripts/composeOtherSitemaps.ts

```diff
@@ -0,0 +1,89 @@
+import fs from 'fs-extra';
+import fetch from 'node-fetch';
+import path from 'path';
+
+async function getRemoteSitemapContent(path: string) {
+  const response = await fetch(`https://storybook.js.org${path}`);
+  const content = await response.text();
+  return content;
+}
+
+const OTHER_SITEMAPS = {
+  addons: {
+    async getContent() {
```

Comment:
This approach appears to be ridiculously over-engineered. But it will allow to update these one-by-one to simply fetch a local sitemap file, once an app is in this monorepo.